sbin/build.sh

```diff
@@ -24,7 +24,7 @@ source "$SCRIPT_DIR/common-functions.sh"
 WORKING_DIR=""
 TARGET_DIR=""
 OPENJDK_REPO_NAME=""
-JVM_VARIANT="-server"
+JVM_VARIANT="JVM_VARIANT:-server"
```

Comment:
I believe this should be `JVM_VARIANT=${JVM_VARIANT:-server}` otherswwise it'll be set to the literal string `JVM_VARIANT:-server`

Reply:
You're right. Will fix.